src/pybind/mgr/nfs/ganesha_conf.py

```diff
@@ -208,50 +209,52 @@ def to_dict(self) -> Dict[str, Any]:
 class CephFSFSAL(FSAL):
     def __init__(self,
                  name: str,
-                 user_id: Optional[str] = None,
+                 cmount_path: Optional[str] = None,
+                #  user_id: Optional[str] = None,
                  fs_name: Optional[str] = None,
-                 sec_label_xattr: Optional[str] = None,
-                 cephx_key: Optional[str] = None) -> None:
```

Comment:
As discussed in downstream slack, I think its OK to start with a branch of quick-and-dirty changes, however I think the core types like FSAL/CephFSFSAL can probably continue being fully compatible with all kinds of ganesha configs. Since user_id and cephx_key can already be None, I don't think it is necessary to comment it out.

Also, we can add cmount_path to the end of the args list.